client-src/elements/chromedash-featurelist.ts

```diff
@@ -214,7 +210,7 @@ class ChromedashFeaturelist extends LitElement {
   }
 
   // Directly called from template/features.html
-  filter(val, shouldPushState) {
+  filter(val, shouldPushState = false) {
```

Comment:
This is the change we talked about in the meeting yesterday, and the change should have no functional ramifications.